### ISSUE TITLE
FAB children buttons don't use style from stylesheet

### ISSUE THREAD

**Issue by SrdjanCosicPrica** [GeekyAnts/NativeBase#1618]
## react-native, react and native-base version
```
"react": "16.2.0",
"react-native": "0.52.0",
"native-base": "2.3.7",
```
## Expected behaviour
![simulator screen shot - iphone 5s - 2018-02-21 at 15 41 06](https://user-images.githubusercontent.com/15933115/36486717-7793dfe0-171f-11e8-933b-109942a2fbb0.png)

## Actual behaviour
![simulator screen shot - iphone 5s - 2018-02-21 at 15 42 22](https://user-images.githubusercontent.com/15933115/36486730-7d0815ae-171f-11e8-9fdf-624844002a38.png)

## Steps to reproduce (code snippet or screenshot)
```
import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  miniFab: {
    backgroundColor: 'orange',
    elevation: 4,
    shadowColor: '#000',
    shadowOffset: { width: 0, height: 2 },
    shadowOpacity: 0.4,
  }
});

...

constructor(props) {
    super(props);

    this.state = {
      isFabActive: false
    };

    this.handleFabPress = this.handleFriendsFabPress.bind(this);
}

handleFabPress() {
    this.setState({
      isFabActive: !this.state.isFabActive
    });
}

render() {
    return (
        <Fab
              active={this.state.isFabActive}
              style={this.state.isFabActive
                ? { backgroundColor: 'ghostwhite' }
                : { backgroundColor: 'orange' }}
              onPress={this.handleFabPress}
            >
              {this.state.isFabActive
                ? <Icon ios='ios-close' android='md-close' style={{ color: '#000' }} />
                : <Icon ios='ios-add' android='md-add' style={{ color: '#000' }} />
              }

              <Button style={styles.miniFab}>
                <Icon ios='ios-contacts' android='md-contacts' style={{ color: '#000' }} />
              </Button>
              <Button style={{
                backgroundColor: '#FFE14E',
              }}>
                <Icon ios='ios-contact' android='md-contact' style={{ color: '#000' }} />
              </Button>
              <Button style={{
                backgroundColor: 'green'
              }}>
                <Icon ios='ios-share-alt' android='md-share' style={{ color: '#000' }} />
              </Button>
            </Fab>
    )
}
```
## Is the bug present in both ios and android or in any one of them?
Only tested in IOS.
## Any other additional info which would help us debug the issue quicker.
The style is correct as it work if i set it on the `<Fab>`

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

**Comment by weikilla** [comment]
> Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

how to use 2.3.10? i use npm install , but i got version 2.12.1
